frontend/src/components/common/CustomTable.vue

```diff
@@ -250,6 +265,13 @@ export default {
         return 'Info Warning';
       }
     },
+    getAssignedPen(assignedPen) {
+      if (assignedPen) {
+        return assignedPen;
+      } else {
+        return 'under review';
```

Comment:
Can you please capitalize? `Under review`

Reply:
@arcshiftsolutions I'd be happy to but it is not this way in the user story, it is lower case